src/components/Pagination/README.md

```diff
@@ -27,14 +27,14 @@ const pagination = <Pagination page={1} pageSize={100} total={1000} onUpdate={ha
 
 | Name            | Description                                                                                              |    Type    | Default |
 | :-------------- | :------------------------------------------------------------------------------------------------------- | :--------: | :-----: |
-| className       | HTML `class` attribute                                                                                   |  `string`  |         |
+| className       | `class` HTML attribute                                                                                   |  `string`  |         |
 | compact         | Hides the title for the `First`, `Previous`, and `Next` buttons. Always set to `true` in mobile version. | `boolean`  | `true`  |
 | onUpdate        | Called when the page number or `pageSize` is changed                                                     | `Function` |         |
-| size            | Size of the pagination items. By default, 'l' in mobile version, 'm' in desktop version                  |  `string`  |         |
+| size            | Size of the pagination items. By default, its value is `l` in mobile version and `m`, in desktop version |  `string`  |         |
```

Comment:
Is it really correct?
`its value is 'l'`

Reply:
Is this question about the English wording or about the correctness of the default value?

Reply:
first

Reply:
I rechecked it with the translator and he said "Yes, it is".